src/AppBundle/EventListener/ExceptionListener.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace AppBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\DependencyInjection\Container;
+
+class ExceptionListener
+{
+    /**
+     * @var Container
+     */
+    private $container;
+
+    /**
+     * @param Container $container
+     */
+    public function __construct(Container  $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * @param GetResponseForExceptionEvent $event
+     *
+     * @return JsonResponse
+     */
+    public function onKernelException(GetResponseForExceptionEvent $event)
```

Comment:
body of this method looks like a hell. Please contact with me directly and try to explain what goal you try to achieve here